### DISCUSSION TITLE
User created group with system permissions?

### DISCUSSION THREAD

**Discussion by departet** [requarks/wiki#6303] [Help / Questions]
Hey!
We habe LDAP auth enabled in our instance of wikiJS and want to map a custom group ("wiki admins" for example) to have the same rights as the Administrators default group. System permissions are greyed out tho and i cant add it to the custom group. Is this intended or is there a way to have a custom group with system (all rights) premissions?
Thank you!

**Comment by moe055** [comment]
Hi

**Comment by departet** [comment]
After further searching i just used the API to add system permissions to my group..
